LLama/Native/SafeLlavaModelHandle.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using LLama;
+using LLama.Exceptions;
+
+
+namespace LLama.Native
+{
+    /// <summary>
+    /// A reference to a set of llava model weights
+    /// </summary>
+    public sealed class SafeLlavaModelHandle
+        : SafeLLamaHandleBase
+    {
+
+        internal protected SafeLlavaModelHandle(IntPtr handle)
+            : base(handle, true)
+        {
+        }
+
+        /// <inheritdoc />
+        protected override bool ReleaseHandle()
+        {
+            
+            NativeApi.clip_free(DangerousGetHandle());
+            SetHandle(IntPtr.Zero);
+            return true;
+        }
+
+        /// <summary>
+        /// Load a model from the given file path into memory
+        /// </summary>
+        /// <param name="modelPath"></param>
+        /// <param name="lparams"></param>
+        /// <returns></returns>
+        /// <exception cref="RuntimeError"></exception>
+        public static SafeLlavaModelHandle LoadFromFile(string modelPath, int verbosity )
+        {
```

Comment:
Can you include a copy of [this code](https://github.com/SciSharp/LLamaSharp/blob/master/LLama/Native/SafeLlamaModelHandle.cs#L93-L99) at the start of this method. It provides **much** better error messages if an invalid file path is used.